app/manage-your-appointments/2025/10/pharmacy-visits-what-we-learnt-part-1/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "What we learnt from visiting pharmacies: part 1"
+date: 2025-10-02
+---
+
+We recently visited 10 pharmacies in London to see how they were using Manage Your Appointments just as appointment booking for this year’s seasonal vaccinations opened. 
+
+We did this research for two main reasons: 
+
+1. To help us better understand the feedback we were receiving: we’ve been getting lots of feedback through our survey, in the Q&A during training webinars and via the service desk. This feedback is great for learning what users want and where they’re running into problems, but usually it’s missing the detail required to fully understand the underlying need. 
+
+2. To hear from users unlikely to take part in remote research: up to this point most of the research on MYA has been remote. This means the sample has skewed towards the kind of users who can easily join a pre-scheduled Teams call. As a result, we’ve heard from lots of back office staff for large pharmacy chains, and people in admin roles for Trusts and PCNs. But, we haven’t heard from as many pharmacists or dispensers as we should have, especially given pharmacies account for roughly 95% of MYA’s users.  
+
+## Different approaches to running seasonal vaccinations
+This research helped us get a clearer picture of how pharmacies run their seasonal vaccination services and how, in turn, they set up appointment availability in MYA. Broadly, the pharmacies I visited took one of two approaches. 
+
+### A flexible approach 
+The most common approach pharmacies took was to allow bookings at any time and set up their appointment availability to mirror their opening hours. That meant going into MYA, creating a weekly repeating session for Monday – Friday, adding all the vaccination services they offer, and then setting the start and finish times to match opening hours. More often than not the start and end dates for this covered the whole campaign. Here's how this set up looks once a user reaches the 'Check your answers' screen:
+
+![Screenshot of a MYA webpage with the title 'Check weekly session' with the details: 1 October to 23 December 2025, Monday to Friday, 09:30 to 17:30, one vaccinator, 5-minute appointments, COVID and flu vaccinations, 96 total appointments in the session and a save session button at bottom.](MYA-check-answers.png)
+
+Pharmacies taking this approach typically had lower demand for both seasonal vaccination appointments and other pharmacy services. They also had more capacity, with two pharmacists on site at any time, or at the very least two trained vaccinators. This gave them the breathing room to offer seasonal vaccinations all day, while still being able to take breaks and stay on top of everything else. 
+
+It also meant they had a relaxed attitude to walk-ins and patients ringing to book an appointment, which they would often just note down in a paper diary. As one pharmacist put it:
+> “it’s not manage your appointments, it’s go with the flow”. 
```

Comment:
Love this quote! For block quotes, you don't need to add quote marks as they’re added automatically (so if you add them they'll appear twice)

```suggestion
> it’s not manage your appointments, it’s go with the flow. 
```